hugegraph-core/src/main/java/com/baidu/hugegraph/security/HugeSecurityManager.java

```diff
@@ -321,6 +385,25 @@ private static boolean callFromCaffeine() {
         return callFromWorkerWithClass(CAFFEINE_CLASSES);
     }
 
+    private static boolean callFromBackendSocket() {
+        // Fixed issue #758
+        return callFromMethods(BACKEND_SOCKET);
+    }
+
+    private static boolean callFromBackendThread() {
+        // Fixed issue #758
+        return callFromMethods(THREAD_NEW) &&
+               callFromWorkerWithClass(BACKEND_THREAD);
+    }
+
+    private static boolean callFromEventHubNotify() {
+        return callFromMethod("com.baidu.hugegraph.event.EventHub", "notify");
```

Comment:
add comment: // Fixed issue #758 notify() will create thread when submit task to executor